datasets/stdpopsim_kern.yml

```diff
@@ -0,0 +1,39 @@
+Name: stdpopsim species resources
+Description: Contains all resources (genome specifications, recombination maps, etc.) required for
+    species specific simulation with the stdpopsim package. These resources are originally from a
+	variety of other consortium and published work but are consolidated here for ease of access and
+	use. If you are interested in adding a new species to the stdpopsim resource please raise an
+	issue on the stdpopsim GitHub page to have the necessary files added here.
+Documentation: https://stdpopsim.readthedocs.io/en/latest/catalog.html
+Contact: https://github.com/popsim-consortium/stdpopsim/issues
+ManagedBy: Andrew Kern & Jerome Kelleher
+UpdateFrequency: Data will be added as new species, genome assemblies, and genetic map data for
+    already included species become available.
+Tags:
+  - population genetics
+  - simulation 
+  - recombination maps
+  - genetic maps
+License: Please see the individual datasets compiled here for licensing details and make sure to
+cite the original sources of any elements of this data that you use.
+Resources:

```

Comment:
The Resources section is required; when you've generated the S3 bucket per Friday's email and CloudFormation template, please add Description, ARN (arn:aws:s3:::[bucket-name], Region (i think us-west-2) and Type: S3 Bucket

Reply:
Added